examples/blob.cpp

```diff
@@ -3,7 +3,6 @@
 #include <string>
 #include <vector>
 #include <iostream>
-#include <cassert>
```

Comment:
If you did not add `#include <cassert>` into this file then you don't need to delete it. It is required cause `assert` is used in this file. I asked to only remove includes which were added by you and became unused

Reply:
Sure, but is there any reason to keep these includes around if they're not used by the examples?

Reply:
If you would prefer a separate PR to remove unneeded `includes`, I've updated this one so that the ones in `examples/` are left unchanged.

Reply:
This include is needed: `assert` macro is used 5 times in this source file

Reply:
Hm, something else must have been pulling it in, as the example continued to build without it (both on my machine and in CI).

Reply:
Yes, it builds cause `<cassert>` is included somewhere else in another included header implicitly. But we must not rely on it cause it is not guaranteed by standard. You also can use `std::string` in a source which does not have `#include <string>` but has `#include <map>` on OS X clang but this code will not compile on Linux GCC.